calliope/test/test_io.py

```diff
@@ -80,6 +79,20 @@ def test_solve_save_read_netcdf(self, model):
             model_from_disk = calliope.read_netcdf(out_path)
             # FIXME test for some data in model_from_disk
 
+    def test_save_read_solve_save_netcdf(self, model):
```

Comment:
Is this test actually working with a model that doesn't have `model_run`? Doesn't `model_run` get added back onto the object on reading it back in, if it exists in the netcdf attrs?

Reply:
Maybe getting a new model from running with the backend interface, then saving *that* to file would work as another / replacement test?